src/CodeGeneration/ApiGenerator/Domain/Specification/UrlPart.cs

```diff
@@ -2,33 +2,45 @@
 
 namespace ApiGenerator.Domain.Specification
 {
+
+	//TODO once https://github.com/elastic/elasticsearch/pull/42346 lands 
+	// Rename this type to Deprecation and remove Path duplication
+	public class DeprecatedPath
+	{
+		public string Version { get; set; }
+		public string Path { get; set; }
+		public string Description { get; set; }
+	}
+	
+	
 	public class UrlPart
 	{
 		private string _description;
 
-		public string Argument
+		public string Argument => $"{LowLevelTypeName} {NameAsArgument}";
+
+		public string LowLevelTypeName
 		{
 			get
 			{
 				//TODO treat list with fixed options as Flags Enum
 				switch (Type)
 				{
-					case "int":
+					case "int": //last i checked does not occur on part
```

Comment:
Shorten to "Does not occur on part" as I started hunting for a variable named `i` :)

Reply:
:joy_cat: sorry about that updated